### DISCUSSION TITLE
Structs

### DISCUSSION THREAD

**Discussion by giscus** [karanpratapsingh/portfolio#27] [Comments]
# Structs

In this tutorial, we will learn about structs in Go

https://www.karanpratapsingh.com/courses/go/structs

**Comment by flyrlabs-rch** [comment]
The first example of type SuperHero struct has a field Power; the later ones have Alias

When passing a struct by value, are the member struct's duplicated as well?

Cheers

**Reply by karanpratapsingh** [reply]
Nice catch, I will fix it.
